stdlib/src/commonMain/kotlin/it/unibo/collektive/stdlib/spreading/Spreading.kt

```diff
@@ -70,67 +76,132 @@ inline fun <ID : Any> Aggregate<ID>.distanceTo(
  * This function features *incremental repair*, and it is subject to the *rising value problem*,
```

Comment:
Here you are stating that it is subject to the raising value problem, but should not.
Additionally, you test that it will not occur